WordPressAuthenticator.xcodeproj/xcshareddata/xcschemes/WordPressAuthenticator.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1130"
+   LastUpgradeVersion = "1150"
```

Comment:
Uh... what dis?

Reply:
It updates the pods project file to the latest version. It _should_ fix this warning: 
<img width="324" alt="Screen Shot 2020-07-09 at 1 15 41 PM" src="https://user-images.githubusercontent.com/1062444/87075894-a4d0af00-c1e6-11ea-9b7c-1b87db0ed41a.png">

Reply:
I tried it again. This doesn't actually fix the warning. Will revert.